event.py

```diff
@@ -1,20 +1,25 @@
+"""
+Implementing CalicoBot events
+"""
 import discord
 import datetime
 from discord.ext import commands
-from discord import app_commands
 import time
-import asyncio
-import difflib
-import main
 
-bot = main.bot
+bot = None
```

Comment:
Looks like I need to change the variable name.
From "bot" 
To "BOT"
Please use replace feature in VSCode (Press F3 and click arrow) 
or
Right click at bot variable and click Rename Symbol then type BOT